test/form_builder/general_test.rb

```diff
@@ -117,8 +117,19 @@ def with_custom_form_for(object, *args, &block)
     assert_select 'form input#user_name.string'
   end
 
-  test 'builder generates text areas for text columns' do
+  test 'builder generates text field for text columns' do
     with_form_for @user, :description
+    if @user.respond_to?(:type_for_attribute) &&
+       @user.type_for_attribute('description').type == :string
+      assert_select 'form input#user_description.string'
```

Comment:
Does this means that in Rails 5.1 a text column will return a input field instead of a text area? If so, this is a huge breaking change and we should handle it here and always return a text area for text column instead of changing the test.